### PR TITLE
Homebrew 4.1.0

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Release notes for 4.1.0.

### CODE REVIEW

File: _posts/2023-07-18-homebrew-4.1.0.md [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+  - [This is now the default behaviour so the `HOMEBREW_INSTALL_FROM_API` variable has been removed and is a no-op](https://github.com/Homebrew/brew/pull/14412).
+  - If you wish to opt-out of this behaviour change, you can [`export HOMEBREW_NO_INSTALL_FROM_API=1`](https://github.com/Homebrew/brew/pull/14375). Please investigate [`HOMEBREW_API_AUTO_UPDATE_SECS`](https://github.com/Homebrew/brew/pull/14589) first.
+  - This behaviour is automatically disabled when using [certain commands](https://github.com/Homebrew/brew/pull/14524) or in configurations that [mostly build from source](https://github.com/Homebrew/brew/pull/14480).
+  - [`HOMEBREW_API_DOMAIN` can be set to use mirrors for](https://github.com/Homebrew/brew/pull/14494) [formulae.brew.sh](https://formulae.brew.sh).
+- Homebrew's analytics are now sent both to Google Analytics and our new InfluxDB instance hosted in the EU.
```
Comment (SMillerDev) [inline _posts/2023-07-18-homebrew-4.1.0.md:20]: Didn't we drop Google?

Comment (MikeMcQuaid) [reply _posts/2023-07-18-homebrew-4.1.0.md:20]: draft PR, please ignore 😁

**Review by issyl0 — APPROVED** [review]

**Review by SMillerDev — DISMISSED** [review]

File: _data/locales/ko.yml [hunk 2]
```diff
@@ -29,4 +29,4 @@ ko:
     foot:
       code: Homebrew는 <a href="https://mxcl.github.io/">Max Howell</a>에 의해 만들어졌습니다.
       page: 웹사이트 <a href="https://exomel.com/">Rémi Prévost</a>, <a href="https://mikemcquaid.com">Mike McQuaid</a> and <a href="https://cargocollective.com/danilalo">Danielle Lalonde</a>.
-      translation: 번역 <a href="https://rkjun.wordpress.com/">Juntai Park</a>, <a href="https://github.com/ghmlee">Graham Lee</a>, ChangJoon and <a href="https://github.com/PricelessCode">Matt Lee</a>.
```
Comment (MikeMcQuaid) [inline _data/locales/ko.yml:32]: This link is broken now.

**Review by issyl0 — DISMISSED** [review]

File: _posts/2023-07-20-homebrew-4.1.0.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- [`brew doctor` warns if Homebrew/homebrew-core or Homebrew/homebrew-cask seem to be tapped unnecessarily so you can `brew untap` them to save time and disk space.](https://github.com/Homebrew/brew/pull/15341)
+- [formulae.brew.sh](https://formulae.brew.sh) provides [new analytics for Homebrew Developer Configuration, OS/Architecture/CI, Homebrew Prefixes and Homebrew Versions](https://formulae.brew.sh/analytics/). This is based on existing data that was already gathered.
+- We destroyed (and did not migrate/back up) all of Homebrew's Google Analytics data on 2023-06-16 and [all Google Analytics code has been removed.](https://github.com/Homebrew/brew/pull/15550)
+- Homebrew's analytics are only sent to our InfluxDB instance hosted in the EU.
+  - If you had previously set `HOMEBREW_NO_ANALYTICS` because you didn't like Google Analytics and/or data being sent to the USA: please consider unsetting this and setting `HOMEBREW_NO_GOOGLE_ANALYTICS` instead, allowing analytics data to be sent to our new InfluxDB host. Again, under some circumstances, [you may see a one-time message nudging you do to this](https://github.com/Homebrew/brew/pull/14592).
```
Comment (p-linnane) [inline _posts/2023-07-20-homebrew-4.1.0.md:18]: We should change “USA” to “US” for consistency.

Does `HOMEBREW_NO_GOOGLE_ANALYTICS` do anything now that the code has been removed?

**Review by carlocab — DISMISSED** [review]

File: _posts/2023-07-20-homebrew-4.1.0.md (hunk 3 above)
Comment (MikeMcQuaid) [reply _posts/2023-07-20-homebrew-4.1.0.md]: Done/fixed both!

**Comment by MikeMcQuaid** [comment]
Thanks for review all ❤️

**Pull request merged**